src/incognito.ts

```
@@ -1,107 +1,264 @@
+          return 'Edge'
+        } else if (ua.match(/OPR/)) {
+          return 'Opera'
         }
+        return 'Chrome'
+      } else {
+        return 'Chromium'
+      }
+    }
+
+    function assertEvalToString (value: number): boolean {
+      return value === eval.toString().length
```

Biome (1.9.4)

[error] eval() exposes to security risks and performance issues.

See the MDN web docs for more details.
Refactor the code so that it doesn't need to call eval().

(lint/security/noGlobalEval)